packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -33,6 +33,24 @@ class DevInstall extends NodePackageInstallTask {
   }
 }
 
+/**
+ * Retrive the package manager from the angular.json
+ *
+ * @param tree
+ * @returns package manager as string or undefined
+ */
+function getPackageManagerFromTree(tree: Tree) {
```

Comment:
this is very similar to the code here https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/schematics/src/utility/package-manager-runner.ts#L21, I suppose you don't reuse because it's needed to install `@o3r/schematics` ?

Reply:
Indeed. It's the case for the class above.

Reply:
finally done in https://github.com/AmadeusITGroup/otter/pull/473